app/src/main/java/com/therap/amin/currencyconverter/Constants.java

```diff
@@ -0,0 +1,13 @@
+package com.therap.amin.currencyconverter;
+
+/**
+ * Created by amin on 5/9/16.
+ */
+public class Constants {
+
+    public static final String preferenceKey = "preferenceKey";
```

Comment:
Use variable name in ALL CAPS for "public static final" type variable declaration. Java Convention. Like
```
public static final String PREFERENCE_KEY = "preference_key";
```